spec/std/zlib/reader_spec.cr

```diff
@@ -65,5 +65,15 @@ module Zlib
       slice = Bytes.empty
       reader.read(slice).should eq(0)
     end
+
+    it "should raise bufer error on error (#6575)" do
```

Comment:
buffer

Reply:
Fixed in 5c314e018447ff08a8768321d82f06309b9a4737